packages/fxa-content-server/server/bin/fxa-content-server.js

```diff
@@ -226,10 +229,11 @@ function listen(theApp) {
     };
 
     https.createServer(tlsoptions, app).listen(port);
+    https.createServer(tlsoptions, app).listen(paymentPort);
     app.on('error', catchStartUpErrors);
   } else {
-    port = config.get('port');
     app.listen(port, '0.0.0.0').on('error', catchStartUpErrors);
+    app.listen(paymentPort, '0.0.0.0').on('error', catchStartUpErrors);
```

Comment:
It isn't very clear to me why we needed to make a new server here? Could we get by with re-using the existing server and adding a route to render the new app, or are there security considerations I am missing?

Reply:
In production, this will need to run on a separate domain from FxA proper in order to isolate some third-party JS for payment processing. Seems like we need to run a separate server on a different port to get that effect here.

Reply:
I'd use a separate server too, for ease of understanding as much as anything. Plus seems like it'd give us some flexibility on how we deploy.

Reply:
I agree we just need a separate server – I thought there's be some saving/simplification to having a single server, but that's all disappeared.